### DISCUSSION TITLE
Skip "Choose Template" when no template and app opens from link

### DISCUSSION THREAD

**Discussion by oscarb** [community/community#12155] [Mobile] [answered]
Hi! First off, thanks for a great mobile app! 👏🏻 

So I was looking for some way to get to "Create Issue" for a given repository with only _one click_ from the home screen on my Android device. I couldn't find any way to do so with the GitHub mobile app itself, so instead I tried creating a shortcut with [Tasker](https://play.google.com/store/apps/details?id=net.dinglisch.android.taskerm) with a simple task of opening a URL, like:

https://github.com/octocat/Spoon-Knife/issues/new

When this URL is opened up on my phone (doesn't matter if it's from the Tasker shortcut or from say any messaging app), the GitHub Mobile app starts and shows me the "Choose Template" view (see picture below). 

![image](https://user-images.githubusercontent.com/17575446/155857449-4c8c8733-d18c-4e62-9542-bac5d6893355.png)

This would be nice if there actually was some templates to choose between, but when there isn't this adds an extra step to get to "Create Issue".

**Request:** Whenever the only option in "Choose Template" is "No template", open the view "Create Issue" instead.

Using GitHub Mobile v1.53.0 (641) on a Sony Xperia 5 III (XQ-BQ52) running Android 12

**Comment by stevepopovich** [comment, marked as answer]
Hey Oscar, great suggestion. Passing on to the team for prioritizing. Thanks!
